### PR TITLE
chore(sitemap): Add url time averages to sitemap links

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

We are not including the new links with the urlTimeAverages in the sitemap.

## Description

Adds this to the sitemap to ensure it's properly picked up by Google and other search engines.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
@cadeban could you update this in your PR to include `72h` once this is merged?

**Review by cadeban — APPROVED** [review]
lgtm!

File: web/scripts/generateSitemap.js [hunk 1]
```diff
@@ -25,7 +30,12 @@ function generateSitemap() {
   }
 
   const zoneUrls = Object.keys(zonesConfig.zones)
-    .map((zone) => `<url><loc>https://app.electricitymaps.com/zone/${zone}</loc></url>`)
+    .flatMap((zone) =>
```
Comment (cadeban) [inline web/scripts/generateSitemap.js:33]: nit: Not sure if it matter much, but the `UrlTimeAverages` also get appended to /map (example: https://bbb1b2b8.electricitymaps-staging.pages.dev/map/12mo)

Comment (VIKTORVAV99) [reply web/scripts/generateSitemap.js:33]: I'll make sure to add them 👍🏼

**Pull request merged**